route/include/route_state_worker.h

```diff
@@ -71,6 +71,9 @@ namespace route {
              */
             RouteState get_route_state();
 
+            //convert RouteState state field to unit8_t
+            uint8_t convertRouteStateToInt ( RouteStateWorker::RouteState state ) const;
```

Comment:
I believe the route state enum should implicitly convert to int which would make this function unneeded.